src/v2/Components/Footer/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./Footer"
```

Comment:
And these index files are largely just noise, and have been phased out of Eigen. We should be doing the same here.

Reply:
Useful when you don't want to update any of the import paths.